README.md

```diff
@@ -3,10 +3,10 @@
 This integration lets you add full automatic and price optimized control over Vehicle to grid (V2G) charging. It has a 
 practical local app in [HomeAssistant](https://www.home-assistant.io/) and uses the smart EMS [FlexMeasures](https://flexmeasures.io) for optimized schedules.
 
-The schedules are optimized on EPEX day-ahead energy prices, so this works best with an electricity contract with dynamic (hourly) prices[^1].
+The schedules are optimized on day-ahead energy prices, so this works best with an electricity contract with dynamic (hourly) prices[^1].
 We intend to add optimisation for your solar generation or the CO₂ content of the grid in the near future.
 
-[^1]: For now: only Dutch energy prices, if you have interest in this software and live outside the Netherlands, please [contact us](https://v2g-liberty.eu/) to see what the options are.
+[^1]: For now: most Dutch enrgy suppliers are listed and all European energy prices (EPEX) are available for optimisation. There also is an option to upload your own prices, if you have an interest in thisplease [contact us](https://v2g-liberty.eu/) to see what the options are.
```

Comment:
Missing space between *this* and *please*.

Reply:
Will correct spelling in next PR.